packages/yarnpkg-core/sources/execUtils.ts

```diff
@@ -51,7 +51,10 @@ export async function pipevp(fileName: string, args: Array<string>, {cwd, env =
 
   const child = crossSpawn(fileName, args, {
     cwd: npath.fromPortablePath(cwd),
-    env,
+    env: {
+      ...env,
+      PWD: npath.fromPortablePath(cwd),
```

Comment:
Right now berry always prints `/c/path/to/cwd`, but with this change berry will always print `C:\path\to\cwd`, which also doesn't match node's behaviour.

Node's behaviour is pretty weird by the way, because they print `C:/path/to/cwd` (note the forward slashes). I'm not sure this isn't a bug on their end?

The BASH shell has `/c/path/to/cwd` in both cases, so I'm leaning towards following that behaviour for consistency if we don't follow node's weirdness.

Reply:
Not sure there's a need to "match" anything - we should just aim to have a sensible behavior. At the moment the `cwd` is different from the `PWD` variable, which is quite unexpected. As long as we fix that, I don't think it's a problem to use either of forward and backward slashes.

Reply:
> At the moment the cwd is different from the PWD variable, which is quite unexpected.

They're different in node as well

![image](https://user-images.githubusercontent.com/821510/82730759-76fcdd00-9d02-11ea-950d-c7bce745e225.png)
Git BASH in VS Code (left) and standalone (right)

I'm not sure what's more unexpected: the environment being different when running `yarn node` vs `node` or the fact that `process.cwd()` is different from the `PWD` environment variable.

Funny aside: deno shows `PWD` to be `C:/path/to/cwd` in both environment, at least they're consistent even if the path is a bit weird.